bids/modeling/tests/test_statsmodels.py

```diff
@@ -1,6 +1,9 @@
 from os.path import join
+from os import path
 from itertools import chain
+from graphviz import Digraph
```

Comment:
```suggestion
from nibabel.optpkg import optional_package
graphviz, has_graphviz, _ = optional_package("graphviz")
```

Reply:
Yeah I think this is definitely a better way to do this.